data/lib/src/network/config/end_point.dart

```diff
@@ -31,24 +31,25 @@
 
 import 'package:domain/domain.dart';
 
-class EndPoint {
+class Endpoint {
   static final String rootPath = '/linshare/webservice/rest/user/v2';
+  static final String download = '/download';
   static final ServicePath authentication = ServicePath('/jwt');
 
   static final ServicePath authorizedUser = ServicePath('/authentication/authorized');
   static final ServicePath documents = ServicePath('/documents');
 }
 
 extension ServicePathExtension on ServicePath {
-  String generateEndPointPath() {
-    return EndPoint.rootPath + path;
+  String generateEndpointPath() {
+    return Endpoint.rootPath + path;
   }
 
   String generateAuthenticationUrl(Uri baseUrl) {
-    return baseUrl.origin + generateEndPointPath();
+    return baseUrl.origin + generateEndpointPath();
   }
 
   String generateUploadUrl(Uri baseUrl) {
```

Comment:
how about do the same for downloadUrl?

Reply:
You are right I'll do it

Reply:
Finally, I will do an extension on download Request to generate the URL. Furthermore, `generateUploadUrl` isn't used in the app..